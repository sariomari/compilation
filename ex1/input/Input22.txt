{"M"]]269
{-30
-527

w//7JP;1;
0
"AJ"-/*;
?*/{"""IIfCh"-(305
//]s]k})m0!()}{d1 /*2]!};;DBOwl}
24}}]}!v
}}7?}*/[	HR}9285
(/**/)//
//]AC{-243

}
2
69
nil{/*!S!u};;p(?v}
;*/)-*1
/*D]*/{/*
*/
VLbv*-[8488
nil9563
6
(+/*}!!*/757
]
	[354
)96
 )-//e
](/*!1
]}!]/]pS;
*/+nil]5601
(6433
575
3273
nil}947
}3603
]G[811

[72
80
]}}( nil--++-*
*)**{]6
45
+}7689
6648

"a"8
pk[nil[ 17
//
-//!2)]841
}318
3668
-1
//D;}!I4}8r]+{-*]//(;n6qZ?!5k]!;}n?
	
)6
nil	/*}}]
s;4z
jB?X?}6*/1276
[Q3 rsIA1
)//!j?Um};?(Vp2-]67
/*z!!*/
443
*2
]12
217
[5
+//]

5378
/*)(1uE;8

]!!}!;
?*/6
[[[L7nEpuj9-60
1957
}nil)
nil//}a;"W"+674
+5

hDq022

nilPU{""-/*?
1gAk(;};R


*/{9457
}))))*{[}nil2
*]{*118
]534
]4
nil//
[}y+{+"X"-+{/*80)}h)[3*/29
//
1
+*-/*}}n]]!!}
}gL}!
!?
]2(?p}

]2*///Z;);)?
-}}"wElY"9283
7973
{6
("I"9
nil7883
-0
[59
42
87
/*}(}5;t}(*///;()!){]]}?!];Lk?]nil({{/*?7
;];
!}?01?
*/5552
*+*)468
5
""q	)[(+ 21
	nil931
})*"pQXmM"2
	{"chLL"{"eluqco"//;)//};!nilkwNE rzr0[(21

	}71
*Lh7470
192
//!))}!
)"hTCvH") {Gz"V"""}[}0
-3
/*];.N]??;*//*ao
]?
*/nil]"Obnj"19
-*[275

//?o}9)?
}174
//?;})?}?]m-//
/*?}R6;}!}*/54
(/*;!}?
z}t)iC]?Z0;?;]!!]]*/2097
335
/*(N]3};t?};!D!6W])*/""(
*/*!l!]*/*""//
7768
//?}9!{
() )6391
7107
//["WvIfim"
662
*9
7
*[4790
6
*]*nil""5055
-("Zc"228
)(""}]-"tiuVZ"{{)/*{*/nil*+{17
	
7030
5
)""}-(})""/*?



o}
D}g4*/1
/*
)8{
))q0?X(*/"moCFyIxPoAEMuLczq"lr545
*"oe"70
"RtC""R"38
nil}[/*];FO5!!!]{])Y
!9
9

i)(})o?W?*/{(}/*!
k??!;v]*/(	ZJ[Vq0dCtnilGT//1?r(5];!?d
 nil{[}/*
*///}M?](63
i[
nil{}/*r)v
g;A;;;;
*/{7978
176
nil}[[[()nil7129
//!;{/*!}8!
;?8]2;}(!a]*/nilnil{{0
"xJEUOKiW"""}1
	X9uRW0GSF7y""218
*//E???}(
*{]]292
11
]783
+
[3
)**464
35
)76
nil/*r
8i.;]R]?pg!]r(]z(?e]]
.)*/988
"iuxCi"*[]/**//*
g(};*/+)]447
4
{{26
)//?
*428
)//!A;);}+-1974
+p/*!3;o);]!*/-2833
{(5
( *XBa9657
((8690
//?
-)[{]7249
i"")//6)
{[
/**/
Brm"ej"{84
{21
}-967
)[}28
{"ZxVZMn"+286
]594
])nil-90
92

nil *1
}-*}s"Qhma"	+]**//!)
[{//++605
{7
(-{/*;8c(?*/(90
[ ))6
201
)+[]{60
/*]*/*(	)u}//}(1286
309
{(l	58
a* +365
]22
70
]	"x"({)3
3377
/*?;]
/
*/(+44
]){nil"hBig"	{//093d};]K;}1zu}	(""/*J?;!})
(]!
}ja]*/*
nil"JlM" 2606

{ {[]//;(;1)U1]9z?(93
*	{24
}*4693
nil-)//(]?7028
"pO"-778
20
}nil}nil{}]//}?}]}
""-EF41CQb06
nil)nil 4409
[)-[91
 +//!
70
nil	)[]{*]//O7)]?)37
56
338
	[}[*94
925
45
595
5
2
{/*?m}O.!E*/""//(	*nil}()(][}nil//4;}!.?;!}}
]6663
581
}[(+-2895
]65
Dw)//!?
421
[]474
4729
5426
}{*]}nil2556
669
5751
({"l"nil
//!}!
)-{{	/*}*/""*/*S

R]*/{+//}?;?;?
   )638
""716
*//]({*3607
"DJcnEkjdT"([]

	]74
*+
]0
nil*-
(2
{)39
/*})?})*/(9
-nil*-s}80
/*!
xDO

;
}0*/8083
(((+76
//))H/;21!((;nil)324
[)Z}++)}-(+([]}-/*T*/6936
]///((?/}W-"x"]73
"u"4474
nil4
})-6372
*
}q8500VL5W0-nil6662
 /*?)p*/)[//}}?;8!-//(]B!)(]+//?}]X
[}nil}-2
//??w
{(27
/**/]]nil259
)[