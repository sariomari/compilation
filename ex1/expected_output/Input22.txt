LBRACE[1,1]
STRING("M")[1,2]
RBRACK[1,5]
RBRACK[1,6]
INT(269)[1,7]
LBRACE[2,1]
MINUS[2,2]
INT(30)[2,3]
MINUS[3,1]
INT(527)[3,2]
ID(w)[5,1]
INT(0)[6,1]
STRING("AJ")[7,1]
MINUS[7,5]
LBRACE[10,3]
STRING("")[10,4]
STRING("IIfCh")[10,6]
MINUS[10,13]
LPAREN[10,14]
INT(305)[10,15]
RBRACE[12,1]
LBRACE[12,2]
ID(d1)[12,3]
LBRACK[15,8]
ID(HR)[15,10]
RBRACE[15,12]
INT(9285)[15,13]
LPAREN[16,1]
RPAREN[16,6]
ID(AC)[18,1]
LBRACE[18,3]
MINUS[18,4]
INT(243)[18,5]
RBRACE[20,1]
INT(2)[21,1]
INT(69)[22,1]
NIL[23,1]
LBRACE[23,4]
RPAREN[24,4]
MINUS[24,5]
TIMES[24,6]
INT(1)[24,7]
LBRACE[25,7]
ID(VLbv)[27,1]
TIMES[27,5]
MINUS[27,6]
LBRACK[27,7]
INT(8488)[27,8]
ID(nil9563)[28,1]
INT(6)[29,1]
LPAREN[30,1]
PLUS[30,2]
INT(757)[30,10]
RBRACK[31,1]
LBRACK[32,2]
INT(354)[32,3]
RPAREN[33,1]
INT(96)[33,2]
RPAREN[34,2]
MINUS[34,3]
RBRACK[35,1]
LPAREN[35,2]
PLUS[37,3]
NIL[37,4]
RBRACK[37,7]
INT(5601)[37,8]
LPAREN[38,1]
INT(6433)[38,2]
INT(575)[39,1]
INT(3273)[40,1]
NIL[41,1]
RBRACE[41,4]
INT(947)[41,5]
RBRACE[43,1]
INT(3603)[43,2]
RBRACK[44,1]
ID(G)[44,2]
LBRACK[44,3]
INT(811)[44,4]
LBRACK[46,1]
INT(72)[46,2]
INT(80)[47,1]
RBRACK[48,1]
RBRACE[48,2]
RBRACE[48,3]
LPAREN[48,4]
NIL[48,6]
MINUS[48,9]
MINUS[48,10]
PLUS[48,11]
PLUS[48,12]
MINUS[48,13]
TIMES[48,14]
TIMES[49,1]
RPAREN[49,2]
TIMES[49,3]
TIMES[49,4]
LBRACE[49,5]
RBRACK[49,6]
INT(6)[49,7]
INT(45)[50,1]
PLUS[51,1]
RBRACE[51,2]
INT(7689)[51,3]
INT(6648)[52,1]
STRING("a")[54,1]
INT(8)[54,4]
ID(pk)[55,1]
LBRACK[55,3]
NIL[55,4]
LBRACK[55,7]
INT(17)[55,9]
MINUS[57,1]
RPAREN[58,1]
RBRACK[58,2]
INT(841)[58,3]
RBRACE[59,1]
INT(318)[59,2]
INT(3668)[60,1]
MINUS[61,1]
INT(1)[61,2]
RBRACK[63,1]
PLUS[63,2]
LBRACE[63,3]
MINUS[63,4]
TIMES[63,5]
RBRACK[63,6]
RPAREN[65,1]
INT(6)[65,2]
NIL[66,1]
INT(1276)[69,3]
LBRACK[70,1]
ID(Q3)[70,2]
ID(rsIA1)[70,5]
RPAREN[71,1]
MINUS[72,1]
RBRACK[72,2]
INT(67)[72,3]
INT(443)[74,1]
TIMES[75,1]
INT(2)[76,1]
RBRACK[77,1]
INT(12)[77,2]
INT(217)[78,1]
LBRACK[79,1]
INT(5)[79,2]
PLUS[80,1]
INT(5378)[82,1]
INT(6)[86,4]
LBRACK[87,1]
LBRACK[87,2]
LBRACK[87,3]
ID(L7nEpuj9)[87,4]
MINUS[87,12]
INT(60)[87,13]
INT(1957)[88,1]
RBRACE[89,1]
NIL[89,2]
RPAREN[89,5]
NIL[90,1]
STRING("W")[91,1]
PLUS[91,4]
INT(674)[91,5]
PLUS[92,1]
INT(5)[92,2]
ID(hDq022)[94,1]
ID(nilPU)[96,1]
LBRACE[96,6]
STRING("")[96,7]
MINUS[96,9]
LBRACE[101,3]
INT(9457)[101,4]
RBRACE[102,1]
RPAREN[102,2]
RPAREN[102,3]
RPAREN[102,4]
RPAREN[102,5]
TIMES[102,6]
LBRACE[102,7]
LBRACK[102,8]
RBRACE[102,9]
ID(nil2)[102,10]
TIMES[103,1]
RBRACK[103,2]
LBRACE[103,3]
TIMES[103,4]
INT(118)[103,5]
RBRACK[104,1]
INT(534)[104,2]
RBRACK[105,1]
INT(4)[105,2]
NIL[106,1]
LBRACK[107,1]
RBRACE[107,2]
ID(y)[107,3]
PLUS[107,4]
LBRACE[107,5]
PLUS[107,6]
STRING("X")[107,7]
MINUS[108,1]
PLUS[108,2]
LBRACE[108,3]
INT(29)[108,16]
INT(1)[110,1]
PLUS[111,1]
TIMES[111,2]
MINUS[111,3]
MINUS[120,1]
RBRACE[120,2]
RBRACE[120,3]
STRING("wElY")[120,4]
INT(9283)[120,10]
INT(7973)[121,1]
LBRACE[122,1]
INT(6)[122,2]
LPAREN[123,1]
STRING("I")[123,2]
INT(9)[123,5]
ID(nil7883)[124,1]
MINUS[125,1]
INT(0)[125,2]
LBRACK[126,1]
INT(59)[126,2]
INT(42)[127,1]
INT(87)[128,1]
NIL[132,1]
LPAREN[132,4]
LBRACE[132,5]
LBRACE[132,6]
INT(5552)[136,3]
TIMES[137,1]
PLUS[137,2]
TIMES[137,3]
RPAREN[137,4]
INT(468)[137,5]
INT(5)[138,1]
STRING("")[139,1]
ID(q)[139,3]
RPAREN[139,5]
LBRACK[140,1]
LPAREN[140,2]
PLUS[140,3]
INT(21)[140,5]
ID(nil931)[141,2]
RBRACE[142,1]
RPAREN[142,2]
TIMES[142,3]
STRING("pQXmM")[142,4]
INT(2)[142,11]
LBRACE[144,1]
STRING("chLL")[144,2]
LBRACE[144,8]
STRING("eluqco")[144,9]
ID(nilkwNE)[146,1]
ID(rzr0)[146,9]
LBRACK[146,13]
LPAREN[146,14]
INT(21)[147,1]
RBRACE[149,2]
INT(71)[149,3]
TIMES[150,1]
ID(Lh7470)[150,2]
INT(192)[151,1]
RPAREN[153,1]
STRING("hTCvH")[153,2]
RPAREN[153,9]
LBRACE[153,11]
ID(Gz)[153,12]
STRING("V")[153,14]
STRING("")[153,17]
RBRACE[153,19]
LBRACK[153,20]
RBRACE[153,21]
INT(0)[153,22]
MINUS[154,1]
INT(3)[154,2]
NIL[158,3]
RBRACK[158,6]
STRING("Obnj")[158,7]
INT(19)[158,13]
MINUS[159,1]
TIMES[159,2]
LBRACK[159,3]
INT(275)[159,4]
RBRACE[162,1]
INT(174)[162,2]
ID(m)[164,1]
MINUS[164,2]
INT(54)[167,1]
LPAREN[168,1]
INT(2097)[174,3]
INT(335)[175,1]
STRING("")[177,8]
LPAREN[177,10]
TIMES[178,1]
TIMES[178,10]
STRING("")[178,11]
INT(7768)[179,1]
LPAREN[181,1]
RPAREN[181,2]
RPAREN[181,4]
INT(6391)[181,5]
INT(7107)[182,1]
LBRACK[184,1]
STRING("WvIfim")[184,2]
INT(662)[185,1]
TIMES[186,1]
INT(9)[186,2]
INT(7)[187,1]
TIMES[188,1]
LBRACK[188,2]
INT(4790)[188,3]
INT(6)[189,1]
TIMES[190,1]
RBRACK[191,1]
TIMES[191,2]
NIL[191,3]
STRING("")[191,6]
INT(5055)[191,8]
MINUS[192,1]
LPAREN[192,2]
STRING("Zc")[192,3]
INT(228)[192,7]
RPAREN[193,1]
LPAREN[193,2]
STRING("")[193,3]
RBRACE[193,5]
RBRACK[193,6]
MINUS[193,7]
STRING("tiuVZ")[193,8]
LBRACE[193,15]
LBRACE[193,16]
RPAREN[193,17]
NIL[193,23]
TIMES[193,26]
PLUS[193,27]
LBRACE[193,28]
INT(17)[193,29]
INT(7030)[195,1]
INT(5)[196,1]
RPAREN[197,1]
STRING("")[197,2]
RBRACE[197,4]
MINUS[198,1]
LPAREN[198,2]
RBRACE[198,3]
RPAREN[198,4]
STRING("")[198,5]
INT(1)[203,7]
STRING("moCFyIxPoAEMuLczq")[206,10]
ID(lr)[206,29]
INT(545)[207,1]
TIMES[208,1]
STRING("oe")[208,2]
INT(70)[208,6]
STRING("RtC")[209,1]
STRING("R")[209,6]
INT(38)[209,9]
NIL[210,1]
RBRACE[210,4]
LBRACK[210,5]
LBRACE[215,12]
LPAREN[215,13]
RBRACE[215,14]
LPAREN[217,6]
ID(ZJ)[217,8]
LBRACK[217,10]
ID(Vq0dCtnilGT)[217,11]
NIL[218,2]
LBRACE[218,5]
LBRACK[218,6]
RBRACE[218,7]
INT(63)[220,1]
ID(i)[221,1]
LBRACK[221,2]
NIL[222,1]
LBRACE[222,4]
RBRACE[222,5]
LBRACE[227,3]
INT(7978)[227,4]
INT(176)[228,1]
NIL[229,1]
RBRACE[229,4]
LBRACK[229,5]
LBRACK[229,6]
LBRACK[229,7]
LPAREN[229,8]
RPAREN[229,9]
ID(nil7129)[229,10]
LBRACE[231,1]
ID(nilnil)[234,4]
LBRACE[234,10]
LBRACE[234,11]
INT(0)[234,12]
STRING("xJEUOKiW")[235,1]
STRING("")[235,11]
RBRACE[235,13]
INT(1)[235,14]
ID(X9uRW0GSF7y)[236,2]
STRING("")[236,13]
INT(218)[236,15]
TIMES[237,1]
TIMES[238,1]
LBRACE[238,2]
RBRACK[238,3]
RBRACK[238,4]
INT(292)[238,5]
INT(11)[239,1]
RBRACK[240,1]
INT(783)[240,2]
PLUS[241,1]
LBRACK[242,1]
INT(3)[242,2]
RPAREN[243,1]
TIMES[243,2]
TIMES[243,3]
INT(464)[243,4]
INT(35)[244,1]
RPAREN[245,1]
INT(76)[245,2]
NIL[246,1]
INT(988)[249,5]
STRING("iuxCi")[250,1]
TIMES[250,8]
LBRACK[250,9]
RBRACK[250,10]
PLUS[251,7]
RPAREN[251,8]
RBRACK[251,9]
INT(447)[251,10]
INT(4)[252,1]
LBRACE[253,1]
LBRACE[253,2]
INT(26)[253,3]
RPAREN[254,1]
TIMES[255,1]
INT(428)[255,2]
RPAREN[256,1]
PLUS[257,1]
MINUS[257,2]
INT(1974)[257,3]
PLUS[258,1]
ID(p)[258,2]
MINUS[261,6]
INT(2833)[261,7]
LBRACE[262,1]
LPAREN[262,2]
INT(5)[262,3]
LPAREN[263,1]
TIMES[264,1]
ID(XBa9657)[264,2]
LPAREN[265,1]
LPAREN[265,2]
INT(8690)[265,3]
MINUS[267,1]
RPAREN[267,2]
LBRACK[267,3]
LBRACE[267,4]
RBRACK[267,5]
INT(7249)[268,1]
ID(i)[269,1]
STRING("")[269,2]
RPAREN[269,4]
LBRACE[270,1]
LBRACK[270,2]
ID(Brm)[272,1]
STRING("ej")[272,4]
LBRACE[272,8]
INT(84)[272,9]
LBRACE[273,1]
INT(21)[273,2]
RBRACE[274,1]
MINUS[274,2]
INT(967)[274,3]
RPAREN[275,1]
LBRACK[275,2]
RBRACE[275,3]
INT(28)[275,4]
LBRACE[276,1]
STRING("ZxVZMn")[276,2]
PLUS[276,10]
INT(286)[276,11]
RBRACK[277,1]
INT(594)[277,2]
RBRACK[278,1]
RPAREN[278,2]
NIL[278,3]
MINUS[278,6]
INT(90)[278,7]
INT(92)[279,1]
NIL[281,1]
TIMES[281,5]
INT(1)[281,6]
RBRACE[282,1]
MINUS[282,2]
TIMES[282,3]
RBRACE[282,4]
ID(s)[282,5]
STRING("Qhma")[282,6]
PLUS[282,13]
RBRACK[282,14]
TIMES[282,15]
TIMES[282,16]
LBRACK[283,1]
LBRACE[283,2]
PLUS[284,1]
PLUS[284,2]
INT(605)[284,3]
LBRACE[285,1]
INT(7)[285,2]
LPAREN[286,1]
MINUS[286,2]
LBRACE[286,3]
LPAREN[286,13]
INT(90)[286,14]
LBRACK[287,1]
RPAREN[287,3]
RPAREN[287,4]
INT(6)[287,5]
INT(201)[288,1]
RPAREN[289,1]
PLUS[289,2]
LBRACK[289,3]
RBRACK[289,4]
LBRACE[289,5]
INT(60)[289,6]
TIMES[290,6]
LPAREN[290,7]
RPAREN[290,9]
ID(u)[290,10]
RBRACE[290,11]
LPAREN[291,1]
INT(1286)[291,2]
INT(309)[292,1]
LBRACE[293,1]
LPAREN[293,2]
ID(l)[293,3]
INT(58)[293,5]
ID(a)[294,1]
TIMES[294,2]
PLUS[294,4]
INT(365)[294,5]
RBRACK[295,1]
INT(22)[295,2]
INT(70)[296,1]
RBRACK[297,1]
STRING("x")[297,3]
LPAREN[297,6]
LBRACE[297,7]
RPAREN[297,8]
INT(3)[297,9]
INT(3377)[298,1]
LPAREN[302,3]
PLUS[302,4]
INT(44)[302,5]
RBRACK[303,1]
RPAREN[303,2]
LBRACE[303,3]
NIL[304,1]
STRING("hBig")[304,4]
LBRACE[304,11]
LPAREN[305,2]
STRING("")[305,3]
TIMES[309,7]
NIL[310,1]
STRING("JlM")[310,4]
INT(2606)[310,10]
LBRACE[312,1]
LBRACE[312,3]
LBRACK[312,4]
RBRACK[312,5]
LPAREN[313,1]
INT(93)[313,2]
TIMES[314,1]
LBRACE[314,3]
INT(24)[314,4]
RBRACE[315,1]
TIMES[315,2]
INT(4693)[315,3]
NIL[316,1]
MINUS[316,4]
RPAREN[316,5]
INT(7028)[317,1]
STRING("pO")[318,1]
MINUS[318,5]
INT(778)[318,6]
INT(20)[319,1]
RBRACE[320,1]
NIL[320,2]
RBRACE[320,5]
NIL[320,6]
LBRACE[320,9]
RBRACE[320,10]
RBRACK[320,11]
STRING("")[321,1]
MINUS[321,3]
ID(EF41CQb06)[322,1]
NIL[323,1]
RPAREN[323,4]
NIL[323,5]
INT(4409)[323,9]
LBRACK[324,1]
RPAREN[324,2]
MINUS[324,3]
LBRACK[324,4]
INT(91)[324,5]
PLUS[325,2]
INT(70)[326,1]
NIL[327,1]
RPAREN[327,5]
LBRACK[327,6]
RBRACK[327,7]
LBRACE[327,8]
TIMES[327,9]
RBRACK[327,10]
INT(37)[328,1]
INT(56)[329,1]
INT(338)[330,1]
LBRACK[331,2]
RBRACE[331,3]
LBRACK[331,4]
TIMES[331,5]
INT(94)[331,6]
INT(925)[332,1]
INT(45)[333,1]
INT(595)[334,1]
INT(5)[335,1]
INT(2)[336,1]
LBRACE[337,1]
STRING("")[338,10]
LPAREN[339,1]
TIMES[339,3]
NIL[339,4]
RBRACE[339,7]
LPAREN[339,8]
RPAREN[339,9]
LPAREN[339,10]
RBRACK[339,11]
LBRACK[339,12]
RBRACE[339,13]
NIL[339,14]
RBRACK[340,1]
INT(6663)[340,2]
INT(581)[341,1]
RBRACE[342,1]
LBRACK[342,2]
LPAREN[342,3]
PLUS[342,4]
MINUS[342,5]
INT(2895)[342,6]
RBRACK[343,1]
INT(65)[343,2]
ID(Dw)[344,1]
RPAREN[344,3]
INT(421)[345,1]
LBRACK[346,1]
RBRACK[346,2]
INT(474)[346,3]
INT(4729)[347,1]
INT(5426)[348,1]
RBRACE[349,1]
LBRACE[349,2]
TIMES[349,3]
RBRACK[349,4]
RBRACE[349,5]
ID(nil2556)[349,6]
INT(669)[350,1]
INT(5751)[351,1]
LPAREN[352,1]
LBRACE[352,2]
STRING("l")[352,3]
NIL[352,6]
RPAREN[354,1]
MINUS[354,2]
LBRACE[354,3]
LBRACE[354,4]
STRING("")[354,11]
TIMES[354,13]
LBRACE[357,4]
PLUS[357,5]
RPAREN[358,4]
INT(638)[358,5]
STRING("")[359,1]
INT(716)[359,3]
TIMES[360,1]
RBRACK[361,1]
LPAREN[361,2]
LBRACE[361,3]
TIMES[361,4]
INT(3607)[361,5]
STRING("DJcnEkjdT")[362,1]
LPAREN[362,12]
LBRACK[362,13]
RBRACK[362,14]
RBRACK[364,2]
INT(74)[364,3]
TIMES[365,1]
PLUS[365,2]
RBRACK[366,1]
INT(0)[366,2]
NIL[367,1]
TIMES[367,4]
MINUS[367,5]
LPAREN[369,1]
INT(2)[369,2]
LBRACE[370,1]
RPAREN[370,2]
INT(39)[370,3]
LPAREN[372,3]
INT(9)[372,4]
MINUS[373,1]
NIL[373,2]
TIMES[373,5]
MINUS[373,6]
ID(s)[373,7]
RBRACE[373,8]
INT(80)[373,9]
INT(8083)[378,5]
LPAREN[379,1]
LPAREN[379,2]
LPAREN[379,3]
PLUS[379,4]
INT(76)[379,5]
NIL[381,1]
RPAREN[381,4]
INT(324)[381,5]
LBRACK[382,1]
RPAREN[382,2]
ID(Z)[383,1]
RBRACE[383,2]
PLUS[383,3]
PLUS[383,4]
RPAREN[383,5]
RBRACE[383,6]
MINUS[383,7]
LPAREN[383,8]
PLUS[383,9]
LPAREN[383,10]
LBRACK[383,11]
RBRACK[383,12]
RBRACE[383,13]
MINUS[383,14]
INT(6936)[384,4]
RBRACK[385,1]
MINUS[387,1]
STRING("x")[387,2]
RBRACK[387,5]
INT(73)[387,6]
STRING("u")[388,1]
INT(4474)[388,4]
ID(nil4)[389,1]
RBRACE[390,1]
RPAREN[390,2]
MINUS[391,1]
INT(6372)[391,2]
TIMES[392,1]
RBRACE[393,1]
ID(q8500VL5W0)[393,2]
MINUS[393,12]
ID(nil6662)[393,13]
RPAREN[394,9]
LBRACK[394,10]
MINUS[395,1]
PLUS[396,1]
LBRACK[397,1]
RBRACE[397,2]
NIL[397,3]
RBRACE[397,6]
MINUS[397,7]
INT(2)[397,8]
LBRACE[400,1]
LPAREN[400,2]
INT(27)[400,3]
RBRACK[402,3]
RBRACK[402,4]
ID(nil259)[402,5]
RPAREN[403,1]
LBRACK[403,2]